pyvisa/util.py

```diff
@@ -64,6 +64,7 @@ def read_user_library_path():
 
```

Comment:
Loading the DLL extra paths when reading the user library path could lead to weird situation since it will be called after looking for standard names (visa, visa32, etc). I think it is unlikely that somebody on windows would manage to get in a situation where VISA is not where we expect it but I think it would be cleaner to have a new utility function called in IVIVISALibrary.get_library_paths before scanning for any dll just in case.

Reply:
Good point, I'll look at that.